test/integration/query-interface.test.js

```diff
@@ -380,6 +380,23 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
         values: ['value1', 'value2', 'value3']
       });
     });
+
+    if (dialect === 'postgres') {
```

Comment:
For some reason I was convinced that test have to cover all dialects.
Unfortunately my pr without answer (https://github.com/sequelize/sequelize/pull/10561#issuecomment-496181575) was hanging for a year, I could have done that `if` myself if someone had replied to me in the thread that I can test for just one dialect.

On one side I am happy we merged my fix, on the other I am sad I won't show as the contributor for it although this is a clear c/p of my pr.

Reply:
Hey @mieszko4, `ARRAY(ENUM)` is postgres only feature, so we only need to cover postgres. Sorry that original PR was left in limbo. I could have commented there but 

1. I wasn't sure if author will still respond after a year
2. Main code and test needed some improvement, which would have taken some extra time.

So I improved code and merged it. Thanks for you initial work

Reply:
Hey @sushantdhiman, sure, no problem, I understand. thnx for the explanation and your contribution overall!